src/main/scala/subsystem/PeripheryBus.scala

```diff
@@ -21,14 +21,15 @@ case object PeripheryBusKey extends Field[PeripheryBusParams]
 
 class PeripheryBus(params: PeripheryBusParams)
                   (implicit p: Parameters) extends TLBusWrapper(params, "periphery_bus")
-    with HasTLXbarPhy
-    with HasCrossing {
-  val crossing = params.sbusCrossingType
+    with HasTLXbarPhy {
+
+  protected val sbus_xing = new CrossingHelper(this, params.sbusCrossingType)
+  def crossTLIn(implicit p: Parameters): TLNode  = sbus_xing.crossTLIn
```

Comment:
Again, I think it might be better to make the crossing public with a good name. I understand you need the forwarding functions for backwards compatibility, but maybe we could just leave them in as deprecated?

Reply:
I'm not especially worried about breaking this BusWrapper api compatibility at this juncture because it only affects the two points where they connect to each other.